routes/index.js

```diff
@@ -67,7 +67,7 @@ export default function routes(app, addon) {
             //  Setup all the parameters we need to pass through to our client.
             var propertyExists = approvalObj.statusCode !== 404;
             var allApprovals = (propertyExists ? approvalObj.value.approvedBy : []);
-            var version = (propertyExists ? approvalObj.version.number : null);
+            // var version = (propertyExists ? approvalObj.version.number : null);
```

Comment:
What is this for?

Reply:
I used `npm run lint` in the test job, and this line declaring a variable that never used can't pass the lint.